jquery.kinetic.js

```diff
@@ -226,6 +224,12 @@
         if (self._useTarget(e.target, e) && self.elementFocused){
           return false;
         }
+      },
+      // prevent selection when dragging
+      selectStart: function (e){
+        if (self._useTarget(e.target, e)){
```

Comment:
Yep you could add a handler to customise allowing selection

On Sun, Aug 9, 2015 at 8:18 AM, Tsai KD <notifications@github.com> wrote:

>> @@ -226,6 +224,12 @@
>>          if (self._useTarget(e.target, e) && self.elementFocused){
>>            return false;
>>          }
>> +      },
>> +      // prevent selection when dragging
>> +      selectStart: function (e){
>> +        if (self._useTarget(e.target, e)){
> what I want to do is like this:
> http://jsfiddle.net/tsaikd/4kL3tc5o/
> some text are selectable inside
> if the selectStart behavior doesn't make sense, how about add a new flag to control?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/davetayls/jquery.kinetic/pull/88/files#r36585224